rtf_vis_tool/src/Components/LandingPageGraph.js

```diff
@@ -0,0 +1,160 @@
+import React, {useEffect, useState} from "react";
```

Comment:
Add comment block like in python to *all* js files.
/*
Copyright

file

author
*/